Find a functionally equivalent code:android.content.res.BridgeTypedArray.getDrawable:COMMENT
Method Modifier: public      
Comment:/**
 * Retrieve the Drawable for the attribute at <var>index</var>.  This
 * gets the resource ID of the selected attribute, and uses
 * {@link Resources#getDrawable Resources.getDrawable} of the owning
 * Resources object to retrieve its Drawable.
 *
 * @param index Index of attribute to retrieve.
 *
 * @return Drawable for the attribute, or null if not defined.
 */

@@ -1,14 +1,7 @@
 {
-    if (index < 0 || index >= mResourceData.length) {
+    if (!hasValue(index)) {
         return null;
     }
-    if (mResourceData[index] == null) {
-        return null;
-    }
     ResourceValue value = mResourceData[index];
-    String stringValue = value.getValue();
-    if (stringValue == null || RenderResources.REFERENCE_NULL.equals(stringValue)) {
-        return null;
-    }
     return ResourceHelper.getDrawable(value, mContext);
 }

